### PR TITLE
Alignement with new promptflow packages

### PR DESCRIPTION
**Pull request by singankit**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow-evals/samples/built_in_evaluators.py [hunk 1]
```diff
@@ -13,12 +13,12 @@
...
-deployment_name = "gpt-4"
+deployment_name = "GPT-4-Prod"
 
 project_scope = {
-    "subscription_id": "2d385bf4-0756-4a76-aa95-28bf9ed3b625",
-    "resource_group_name": "rg-name",
-    "project_name": "project-name",
+    "subscription_id": "e0fd569c-e34a-4249-8c24-e8d723c7f054",
+    "resource_group_name": "resource-group",
+    "project_name": "project-name
```
Comment (nagkumar91) [inline src/promptflow-evals/samples/built_in_evaluators.py:21]:
```suggestion
    "project_name": "project-name"

```

**Review by nagkumar91 — APPROVED** [review]

**Review by luigiw — DISMISSED** [review]

**Comment by ninghu** [comment]
Do you want to update dev-setup/main.py to include promptflow-evals?

https://github.com/microsoft/promptflow/blob/6b9c5c6f1d62da1a7b9d307891902c6fac8af0c1/scripts/dev-setup/main.py#L15

File: .github/workflows/promptflow-evals-unit-test.yml [hunk 2]
```diff
@@ -52,7 +52,7 @@ jobs:
           path: ${{ env.WORKING_DIRECTORY }}
       - name: install promptflow-evals from wheel
         # wildcard expansion (*) does not work in Windows, so leverage python to find and install
-        run: poetry run pip install $(python -c "import glob; print(glob.glob('promptflow_evals-*.whl')[0])")
+        run: poetry run pip install --pre $(python -c "import glob; print(glob.glob('promptflow_evals-*.whl')[0])")
```
Comment (ninghu) [inline .github/workflows/promptflow-evals-unit-test.yml:55]: I think it installs from local built wheel. do we really need `--pre` here?

Comment (singankit) [reply .github/workflows/promptflow-evals-unit-test.yml:55]: This is for dependencies right now. promptflow-core is published only as a beta version package right now.

Comment (singankit) [reply .github/workflows/promptflow-evals-unit-test.yml:55]: Once we have an official release version this can be removed.

**Review by nagkumar91 — DISMISSED** [review]

File: src/promptflow-evals/samples/built_in_evaluators.py [hunk 3]
```diff
@@ -114,7 +115,7 @@ def run_safety_evaluators():
 
 
 def run_qa_evaluator():
-    qa_eval = QAEvaluator(model_config=model_config, deployment_name="gpt-4")
+    qa_eval = QAEvaluator(model_config=model_config, deployment_name=deployment_name)
```
Comment (ninghu) [inline src/promptflow-evals/samples/built_in_evaluators.py:118]: I sort of remember that Clement mentioned below needs to be changed as well.

from promptflow.entities import AzureOpenAIConnection

Comment (ninghu) [reply src/promptflow-evals/samples/built_in_evaluators.py:118]: I meant to say this change is missed for unit-test files

File: scripts/dev-setup/main.py [hunk 4]
```diff
@@ -19,6 +19,7 @@
     "promptflow-azure",
     "promptflow[azure]",
     "promptflow-tools",
+    # "promptflow-evals",
```
Comment (ninghu) [inline scripts/dev-setup/main.py:22]: any reason why commented?

**Review by ninghu — DISMISSED** [review]

File: scripts/dev-setup/main.py (hunk 4 above)
Comment (singankit) [reply scripts/dev-setup/main.py]: I was trying out something but have changed it back

**Review by nagkumar91 — APPROVED** [review]

**Review by ninghu — APPROVED** [review]

**Pull request merged**